pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -58,8 +59,8 @@ type vmValidator struct {
 	vmCache       ctlkubevirtv1.VirtualMachineCache
 	vmiCache      ctlkubevirtv1.VirtualMachineInstanceCache
 	nadCache      ctlcniv1.NetworkAttachmentDefinitionCache
-
-	rqCalculator *resourcequota.Calculator
+	settingCache  ctlharvesterv1.SettingCache
```

Comment:
do we need to have `settingCache` in the `vmValidator` type? the only time it seems to be needed is in the resourcequota calculator

Reply:
settingCache is used to fetch overhead ratio, to compute overhead, this is forgotten in v140